locales/templates/app.pot

```diff
@@ -776,3 +778,40 @@ msgctxt "tabs_tray_action_erase"
 msgid "Erase browsing history"
 msgstr ""
 
+#. If the user decides to open the current page in another browser and if
+#. Firefox is not installed then we show this       item in the list of
+#. browsers. Clicking it will open the app store installed on the device
+#. (usually that's Google Play).
+msgctxt "download_firefox"
+msgid "Download Firefox"
+msgstr ""
+
+#. Text shown in the "Your rights" screen.            %1$s will be replaced
+#. with the name of the app (e.g. Firefox Focus)            %2$s will be
+#. replaced with a URL linking to the MPL (e.g.
+#. https://www.mozilla.org/en-US/MPL/)            %3$s will be replaced with a
+#. URL linking to the trademark policy (e.g.
+#. https://www.mozilla.org/foundation/trademarks/policy/)            %4$s will
+#. be replaced with a URL linking to the GPL            %5$s will be replaced
+#. with a URL linking to the tracking protection wiki page (e.g.
+#. https://wiki.mozilla.org/Security/Tracking_protection#Lists)            %6§s
+#. will be replaced with a URL linking to the list of licenses used by the
+#. dependencies.
+msgctxt "your_rights_content"
+msgid ""
+"<p>%1$s is free and open source software made by Mozilla and other "
+"contributors.</p>\n"
+"<p>%1$s is made available to you under the terms of the <a "
+"href=\"%2$s\">Mozilla Public License</a> and other open source "
+"licenses.</p>\n"
+"<p>You are not granted any rights or licenses to the trademarks of the "
+"Mozilla Foundation or any party, including the Mozilla, Firefox or %1$s "
+"names or logos. Additional information may be found <a "
+"href=\"%3$s\">here</a>.</p>\n"
+"<p>Additional source code for %1$s is available under various other free "
+"and open source <a href=\"%6$s\">licenses</a>.</p>\n"
+"<p>%1$s also uses blocklists provided by Disconnect, Inc. as separate and"
+" independent works under the <a href=\"%4$s\">GNU General Public License "
+"v3</a>, and available <a href=\"%5$s\">here</a>.</p>"
+msgstr ""
```

Comment:
Ugh, this is horrible from a localization point of view. Really easy to break markup, and changing a comma in English would invalidate the entire translation.

Can we split this up, one string per paragraph, and drop some markup in the process?

```
string 1 = "%1$s is free and open source software made by Mozilla and other contributors."
string 2 = "%1$s is made available to you under the terms of the <a href=\"%2$s\">Mozilla Public License</a> and other open source licenses."

etc.
```

Reply:
I was following the pattern of ``about_content`` here. But yeah, I can break this up. :)

Reply:
That one is slightly different, but we should have cleaned up a bit of HTML anyway :(